src/others/fibonacci.js

```diff
@@ -18,22 +18,19 @@
 (function (exports) {
   'use strict';
 
-  function fibonacci (n) {
-    if (n > 97) {
-      throw 'Input too large, results in inaccurate fibonacci value.';
+  function fibonacci(number) {
+    if (number <= 1) {
+      return number;
     }
-    var n1 = 0;
-    var n2 = 1;
-    var aux;
 
-    while (n > 0) {
-      aux = n1;
-      n1 = n2;
-      n2 += aux;
-      n = n - 1;
+    function recursion(length, originalLength, previous, next) {
```

Comment:
This may work a bit faster only with ES2015 and above when there's a TCO.

Reply:
I'd prefer to not introduce such change, there's no clear performance benefit.

Reply:
Yes, but it's still easier to read because there is less code 😊